### DISCUSSION TITLE
Re-encode audio

### DISCUSSION THREAD

**Discussion by jkyselak** [blakeblackshear/frigate#911] [Ask A Question]
I have one camera that has a non-AAC audio codec (G711 or G726). The Frigate documentation states "If your camera doesn't support a compatible format for RTMP, you can use the ffmpeg args to re-encode it on the fly at the expense of increased CPU utilization".
I tried to figure out the args but was not successful, I was also curious about how much of a CPU load increase this will cause.

Answering my own question: "-c:v copy -c:a aac" args will copy the video stream and convert the audio to AAC. CPU increase - at least in my case - is negligible.

**Comment by TheSleepySlee** [comment]
Thanks for the solution :)
Adding the video output arg part is important, as at first I only added "-c:a aac" and the CPU usage skyrocketed.
